Documentation/connectors-configuration.md

```diff
@@ -0,0 +1,84 @@
+# Configuring Connectors
+
+Connectors connect Dex to authentication providers. Dex needs to have at least one connector configured so that users can log in.
+
+## Configuration Format
+
+The Dex connector configuration format is a JSON array of objects, each with an ID and type, in addition to whatever other configuration is required, like so:
+
+```
+[
+    {
+        "id": "local",
+        "type": "local"
+    }, {
+        "id": "Google",
+        "type": "oidc",
+        ...<<more config>>...
+    } ,
+]
+```
+
+The additional configuration is dependent on the specific type of connector.
+
+### `local` connector.
+
+The `local` connector allows email/password based authentication hosted by dex itself. It is special in several ways:
+
+* There can only be one `local` connector in your configuration.
+* The id must be `local`
+* No other configuration is required
+
+When the `local` connector is present, users can authenticate with the "Log in With Email" button on the authentication screen.
+
+The configuration for the local connector is always the same; it looks like this:
+
+```
+    {
+        "id": "local",
+        "type": "local"
+    }
+```
+
+### `oidc` connector
+
+The `oidc` connector lets users authenticate with other OIDC providers - including other dex servers! In addition to `id` and `type`, the `oidc` connector takes the following additional fields:
```

Comment:
Maybe we don't confuse people with the "including other dex servers" piece...

Replace "the `oidc` connector" with "the connector config" as to not conflate `id`/`type`/`oidc`.